content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,262 @@
+not deploy an application based on the application status.
+
+You can use the information below to decide which of the applications you should
+deploy to your production system, and adjust the manifest accordingly.
+
+Application status indicators for Kubeflow:
+
+* **Stable** means that the application complies with the 
+  [criteria](https://github.com/kubeflow/community/blob/master/guidelines/application_requirements.md)
+  to reach application version 1.0, and that the Kubeflow community has deemed 
+  the application stable for this release of Kubeflow.
+* **Beta** means that the application is working towards a version 1.0 release
+  and its maintainers have communicated a timeline for satisfying the criteria
+  for the stable status.
+* **Alpha** means that the application is in the early phases of 
+  development and/or integration into Kubeflow.
+
+The **application version** in the table reflects the application version in
+the manifest at the time when Kubeflow {{% kf-latest-version %}} was
+released. This is therefore the default version of the application that you
+receive when you deploy Kubeflow {{% kf-latest-version %}}. Some applications 
+may release later versions that you can choose to install into your Kubeflow
+deployment. If you need a later version of a specific application, refer to the
+documentation for that application.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Application</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
+        <th>Application version in Kubeflow {{% kf-latest-version %}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="/docs/components/central-dash/overview/">Central 
+          dashboard: Kubeflow UI</a>
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/chainer/">Chainer operator</a>
+        (<a href="https://github.com/kubeflow/chainer-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/hyperparameter-tuning/overview/">Hyperparameter
+          tuning: Katib</a>
+          (<a href="https://github.com/kubeflow/katib">GitHub</a>)
+          </td>
+        <td>Beta</td>
+        <td>v1alpha3</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/kfserving/">KFServing</a>
+          (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>v0.2.2</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+          (<a href="https://github.com/kubeflow/metadata">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
```

Comment:
/cc @zhenghuiwang Please identify the version of Metadata release to be put in here. if there isn't going to be a version update, please list the current version.

Reply:
We can put `0.2.1` as the version. This is the third release for metadata, but it doesn't have any major update.

Reply:
Done.